http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/db/AppServiceDbQueriesSpec.scala

```diff
@@ -1,442 +0,0 @@
-    )
-
-    // delete nodepool for deleted app
-    dbFutureValue(nodepoolQuery.markAsDeleted(savedNodepool.id, destroyedDate)) shouldBe 1
-    val listAppsWithDeleted2 = dbFutureValue {
-      KubernetesServiceDbQueries.listFullApps(Some(cluster1.cloudContext), includeDeleted = true)
-    }
-    listAppsWithDeleted2.length shouldEqual 1
-    listAppsWithDeleted2.head.nodepools.length shouldEqual 1
-    val deletedNodepool = listAppsWithDeleted2.head.nodepools.head
-    deletedNodepool.copy(apps = List()) shouldEqual savedNodepool.copy(
-      status = NodepoolStatus.Deleted,
-      auditInfo = savedNodepool.auditInfo.copy(destroyedDate = Some(destroyedDate)),
-      apps = List()
-    )
-    deletedNodepool.apps.length shouldEqual 1
-    nodepool.apps.head shouldEqual app1.copy(status = AppStatus.Deleted,
-                                             auditInfo = app1.auditInfo.copy(destroyedDate = Some(destroyedDate))
-    )
-  }
-
-  it should "not list deleted apps when includeDelete is false" in isolatedDbTest {
-    val cluster1 = makeKubeCluster(1).save()
-    val nodepool1 = makeNodepool(1, cluster1.id).save()
-    val app1 = makeApp(1, nodepool1.id).save()
-    val app2 = makeApp(2, nodepool1.id).save()
-
-    val destroyedDate = Instant.now()
-    dbFutureValue(appQuery.markAsDeleted(app1.id, destroyedDate)) shouldBe 1
-
-    val listAppsWithoutDeleted = dbFutureValue {
-      KubernetesServiceDbQueries.listFullApps(Some(cluster1.cloudContext), includeDeleted = false)
-    }
-    listAppsWithoutDeleted.length shouldEqual 1
-    listAppsWithoutDeleted.head.nodepools.length shouldEqual 1
-    val nodepool = listAppsWithoutDeleted.head.nodepools.head
-    nodepool.apps.length shouldEqual 1
-    nodepool.apps.head shouldEqual app2
-  }
-
-  it should "save cluster when one doesn't exist for project" in isolatedDbTest {
-    val makeCluster1 = makeKubeCluster(1)
-    val saveCluster1 = Some(makeCluster1)
-      .map(c =>
-        SaveKubernetesCluster(
-          c.cloudContext,
-          c.clusterName,
-          c.location,
-          c.region,
-          c.status,
-          c.ingressChart,
-          c.auditInfo,
-          DefaultNodepool.fromNodepool(c.nodepools.headOption.get),
-          c.workspaceId
-        )
-      )
-      .get
-
-    val saveResult = dbFutureValue(KubernetesServiceDbQueries.saveOrGetClusterForApp(saveCluster1))
-
-    val x = saveResult match {
-      case _: ClusterExists       => false
-      case _: ClusterDoesNotExist => true
-    }
-    x shouldEqual true
-    saveResult.minimalCluster shouldEqual makeCluster1
-  }
-
-  it should "error if cluster exists and is creating on save" in isolatedDbTest {
-    val makeCluster1 = makeKubeCluster(1).copy(status = KubernetesClusterStatus.Provisioning)
-    val makeCluster2 = makeKubeCluster(2).copy(status = KubernetesClusterStatus.Precreating)
-
-    makeCluster1.save()
-    makeCluster2.save()
-
-    val saveCluster1 = Some(makeCluster1)
```

Comment:
The pattern `Some(x).map(f).get` pattern was repeated a few places which made me chuckle/ 🤦